Gemfile

```diff
@@ -21,17 +21,19 @@ gem 'jbuilder', '~> 2.7'
 # gem 'redis', '~> 4.0'
 # Use Active Model has_secure_password
 # gem 'bcrypt', '~> 3.1.7'
-gem 'devise_token_auth'
-gem 'rack-cors'
-gem 'httparty'
-gem 'dotenv'
+# gem 'devise_token_auth'
```

Comment:
What's with the comment here?

Reply:
I had problems with Travis, he did not pass the tests and he showed me errors with devise, so I was making changes in some parts to find a solution, I missed that commented gem, it is already solved